controllers/cluster_status.go

```diff
@@ -165,7 +165,7 @@ func (r *ClusterReconciler) getNodes(ctx context.Context) (map[string]corev1.Nod
 	return data, nil
 }
 
-func (r *ClusterReconciler) getManagedPods(
+func (r *ClusterReconciler) getManagedInstances(
```

Comment:
we cannot scope the List query to pick up only the instance pods because we don't keep track in an unambiguous way of the instance at the pod creation.

The instance pods are identified only when the label role is applied but this happens after the creation.

This is why we are adding the new label on the instance pods, but it will make time until we can safely use it in our code due to backward compatibility. 

We should also think of a function that reconciles the pod labels with the new value that we are adding